shell/platform/darwin/ios/ios_context.h

```diff
@@ -133,10 +140,13 @@ class IOSContext {
 
   virtual std::shared_ptr<impeller::Context> GetImpellerContext() const;
 
+  int get_msaa_samples() const { return msaa_samples_; }
```

Comment:
Should it be `int GetMSAASamples()`

Reply:
Done (but as `GetMsaaSamples`. I think our style guide does want this to be `get_msaa_samples` but that style isn't used elsewhere in this file.